charts/pulsar-operator/templates/bookkeeper-operator/rbac.yaml

```diff
@@ -18,16 +18,22 @@
 #
 
 {{- if .Values.components.bookkeeper }}
-kind: {{ template "pulsar.bookkeeperRoleKind" . }}
+{{- $namespaces := splitList "," .Values.watchNamespace }}
+{{- if .Values.pulsar.serviceAccount.clusterRole }}
+{{- $namespaces = list (first $namespaces) }}
+{{- end }}
```

Comment:
If `.Values.pulsar.serviceAccount.clusterRole` is true, only need to create one clusterRole, so I add this line. Or it may create duplicated resources.